R/parse_authors.R

```diff
@@ -0,0 +1,62 @@
+#' Parse the `Author` field from a DESCRIPTION file
+#' 
+#' Parse the `Author` field from a DESCRIPTION file into a `person` object
+#'
+#' @param author_string A character containing the `Author` or `Maintainer` 
+#'   field from a  `DESCRIPTION` file
+#'
+#' @return A `person` object
+#' 
+#' @importFrom utils as.person
+#' 
+#' @examples
+#' # Read from a DESCRIPTION file directly
+#' utils_description <- system.file("DESCRIPTION", package = "utils")
+#' utils_authors <- read.dcf(utils_description, "Author")
```

Comment:
I just spent five minutes figuring out why I couldn't get this for **linelist**, and now I realized that it is single here and plural for `@R` 😅